packages/react/src/hooks/useToggleChat.ts

```diff
@@ -0,0 +1,32 @@
+import { setupChatToggle } from '@livekit/components-core';
+import { useLayoutContext } from '../context';
+import { mergeProps } from '../mergeProps';
+import * as React from 'react';
+
+/** @public */
+export interface UseToggleChatProps {
+  props: React.ButtonHTMLAttributes<HTMLButtonElement>;
+}
+
+/** @public */
+export function useToggleChat({ props }: UseToggleChatProps) {
```

Comment:
we generally have the semantics of `use*Toggle` so I'd suggest renaming this to `useChatToggle` which aligns better with the component name, if we make this public

Reply:
good catch 👍